.circleci/config.yml

```diff
@@ -74,6 +74,7 @@ jobs:
       - run:
           name: 🧹 Run Rubocop
           command: |
+            bundle install
```

Comment:
Is this line needed ?

Reply:
I think this is not needed. Should i remove & commit ?